Feature/13/project structure

Alan-day opened the pull request:
We created boilerplate code for all pages and provided routes for each one of them.

Alan-day committed:
boilerplate code and structure for Client Staff and Welcome Home pages

pilahr committed:
Created boiler plate for Client,Booking,Setting pages and routes added

Alan-day committed:
created all routes for pages

olirob93 reviewed (changes requested):
You have duplicated css and scss files in every page. You should only have 2 files in each page a jsx file and an scss file.

Check your vs code extensions as there may be one on that is creating an extra css folder. 

You have a welcome page but with no route to it. You need to change this.

pilahr committed:
De
leted css files, imported all the components that needed for the pag
es, added the route for home page and re-route the WelcomePage to de
fault and remove the router tag.

pilahr committed:
deleted unwanted css files, add routes to home page and WelcomePage and imported components for every pages to App.jsx

pilahr committed:
Added insignificant changes to update 'De' comment

pilahr committed:
Deleted the unwanted css files in styles folder

pilahr committed:
Deleted ClientDelete, ClientSearch and ClientSorting pages

pilahr committed:
Merge branch 'main' of https://github.com/nology-tech/Tahiti-Client-Project into feature/13/project-structure

olirob93 reviewed (approved):
It looks all good
